src/register.js

```diff
@@ -41,9 +39,55 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-export default function Register() {
+export default function Register(props) {
+    const useInput = initialValue => {
```

Comment:
you already using it on login, it will be good idea to move this hook to own file and import it in both places you are using it.